CONTRIBUTING.md

```diff
@@ -4,8 +4,31 @@ Cycloid Team is glad to see you contributing to this project ! In this document,
 
 ## Reporting an issue
 
+### Issues
+
 When you find a bug in Terracognita, it should be reported using [GitHub issues](https://github.com/cycloidio/terracognita/issues). Please define key information like your Operating System (OS), Terracognita origin (docker or from source) and finally the version you are using.
 
+### Issue Types
+
+There are 6 types of issues:
+
+- `enhancement`: These track specific feature requests and ideas until they are complete. They can evolve from a `specification` or they can be submitted individually depending on the size.
+- `specification`: These track issues with a detailed description, this is like a proposal.
+- `bug`: These track bugs with the code
+- `docs`: These track problems with the documentation (i.e. missing or incomplete)
+- `maintenance`: These tracks problems, update and migration for dependencies / third-party tools
+- `refactoring`: These tracks internal improvment with no direct impact on the product
+
+## Opening a Pull Request (PR)
+
+### PR status
+
+There are 3 types of status:
+
+- `need review`: this status must be set when you feel confident with your submission
+- `in progress`: some important change has been requested on your submission, so you can toggle from `need review` to `in progress`
+- `under discussion`: it's time to take a break, think about this submission and try to figure out how we can implement this or this
```

Comment:
All 3 can go on issues basically, the "under discussion" is actually one that fits issues - not really PR